src/entities/document/components/DocumentNewForm/CoverLetterNewForm.stories.tsx

```
@@ -0,0 +1,132 @@
+import { useState } from "react";
+
+import { http, HttpResponse } from "msw";
+
+import { CoverLetterNewForm } from "./CoverLetterNewForm";
+import type { Meta, StoryObj } from "@storybook/react-vite";
+import { within, userEvent, waitFor, expect } from "@storybook/test";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+const withReactQuery = (Story: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)